pkg/apis/common/v0alpha1/zz_generated.openapi.go

```diff
@@ -19,58 +18,6 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 	return map[string]common.OpenAPIDefinition{
 		"github.com/grafana/grafana/pkg/apis/common/v0alpha1.ObjectReference": schema_pkg_apis_common_v0alpha1_ObjectReference(ref),
 		"github.com/grafana/grafana/pkg/apis/common/v0alpha1.Unstructured":    Unstructured{}.OpenAPIDefinition(),
-		"k8s.io/apimachinery/pkg/apis/meta/v1.APIGroup":                       schema_pkg_apis_meta_v1_APIGroup(ref),
```

Comment:
We cover these standard openapi defs in this helper [here](https://github.com/grafana/grafana/blob/main/pkg/services/grafana-apiserver/openapi.go#L29), so deleting their repetition in each of the group versions defs should not impact the actual outcome.